package.json

```diff
@@ -6,6 +6,8 @@
   "scripts": {
     "dev:server": "cross-env NODE_ENV=development npm run ts:exec server/index.ts",
     "dev": "nodemon",
+    "docker:db": "docker-compose exec pg psql -U postgres -d dev-directory",
```

Comment:
This command runs the command `psql -U postgres -d dev-directory` (which just connects to the local db) _on_ the docker container by name, `pg`